packages/codemirror/src/editor.ts

```diff
@@ -871,6 +871,210 @@ class CodeMirrorEditor implements CodeEditor.IEditor {
 export
 namespace CodeMirrorEditor {
   /**
+   * The options used to initialize a code mirror editor.
+   */
+  export
+  interface IOptions extends CodeEditor.IOptions {
```

Comment:
I like this approach of extending the `CodeEditor.IOptions` rather than having a second one. One question though, for options that we name differently `wordWrap` and that are repeated in the CM config, are we telling people that they have to pass in the general name and not the CM one?